app/src/main/java/com/rokt/roktdemo/ui/layouts/ScanQrViewModel.kt

```diff
@@ -54,26 +58,51 @@ class ScanQrViewModel @Inject constructor(
     fun executePreview(embeddedWidget: WeakReference<Widget>?) {
         val scannedData = _state.value.data?.scannedData
         scannedData?.let { data ->
-            val attributes = hashMapOf(
-                ATTRIBUTE_IS_DEMO to true.toString(),
-                ATTRIBUTE_LAYOUT_ID to data.previewId,
-                ATTRIBUTE_VERSION_ID to data.versionId,
-                ATTRIBUTE_CREATIVE_ID to data.creativeIds.joinToString(separator = ",")
-            )
+            val attributes = buildMap {
+                this[ATTRIBUTE_IS_DEMO] = true.toString()
+                this[ATTRIBUTE_CREATIVE_ID] = data.creativeIds.joinToString(separator = ",")
```

Comment:
Actually, this one won't be required as well, as we defined in demoConfig

Reply:
ya, I thought so. Anyway we can update it in next version.